modules/ossm-understanding-versions.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+// * service_mesh/v1x/upgrading-ossm.adoc  ???
+// * service_mesh/v2x/upgrading-ossm.adoc
+// * service_mesh/v2x/ossm-troubleshooting.adoc
+
+[id="ossm-versions_{context}"]
+= Understanding Service Mesh versioning
```

Comment:
Not sure this is a troubleshooting topic, as it seems like general information that you'd want to know when installing service mesh... I wonder if it'd be better to have this in the installation section or around pre-reqs?

Reply:
I was considering also including this topic either in the installation or upgrade section.